Source/JavaScriptCore/heap/Strong.h

```diff
@@ -157,8 +172,31 @@ namespace WTF {
 
 template<typename T> struct VectorTraits<JSC::Strong<T>> : SimpleClassVectorTraits {
     static constexpr bool canCompareWithMemcmp = false;
+#if ENABLE(REFTRACKER)
+    static constexpr bool canInitializeWithMemset = false;
+    static constexpr bool canMoveWithMemcpy = false;
+#endif
 };
 
-template<typename P> struct HashTraits<JSC::Strong<P>> : SimpleClassHashTraits<JSC::Strong<P>> { };
+template<typename P> struct HashTraits<JSC::Strong<P>> : SimpleClassHashTraits<JSC::Strong<P>> {
+#if ENABLE(REFTRACKER)
+    using S = JSC::Strong<P>;
+    static constexpr bool emptyValueIsZero = false;
```

Comment:
if constexpr (Traits::emptyValueIsZero)
        return reinterpret_cast_ptr<ValueType*>(static_cast<char*>(HashTableMalloc::zeroedMalloc(size * sizeof(ValueType))));
        
  If we allocate a bucket but don't call Strong(), then set it later, then the ref tracker will never find out about it. To be honest, it can probably be made to work without this, but I was having a very difficult time figuring out why I was seeing a ref that was created but never freed, so I just tried to be conservative.